README.rst

```diff
@@ -1,6 +1,9 @@
 |logo| Pootle
 =============
 
+Pootle is in the process of looking for potential maintainers to take over. Until then, users can use `Weblate <https://github.com/WeblateOrg/weblate>`_ as an alternative.
+------------------
```

Comment:
in rst the underline needs to be at least as long as the sentence - usually best to make them the same length

Reply:
- I modified the document. To be honest I had never used reStructuredText before.